code/game/atoms_movable.dm

```diff
@@ -109,6 +202,8 @@
 			O.Check()
 	if (orbiting)
 		orbiting.Check()
+	if(isturf(loc) && forced_gravity_by_turf && (loc != forced_gravity_by_turf))
+		forced_gravity_by_turf.reset_forced_gravity_atom(src)

```

Comment:
Perhaps it should have a global var (so only one report of it per round) and a debug report to state this occured with some sort of trace so we can debug in the future?